components/ws-manager-bridge/src/prebuild-updater.ts

```diff
@@ -39,6 +39,7 @@ export class PrebuildUpdater {
         const span = TraceContext.startSpan("updatePrebuiltWorkspace", ctx);
         try {
             const prebuild = await this.workspaceDB.trace({ span }).findPrebuildByWorkspaceID(status.metadata!.metaId!);
+            const workspace = await this.workspaceDB.trace({ span }).findById(workspaceId);
```

Comment:
It's a bit weird that PrebuiltWorkspace has a projectId but no organizationId... :grimacing: But it is what it is I guess.